models/comments.model.js

```diff
@@ -0,0 +1,20 @@
+const db = require("../db/connection.js");
+const { checkArticleExists } = require("../models/fetchArticles.model.js");
+
+exports.insertComment = async (comment) => {
+  const { username, body, article_id } = comment;
+  if (await checkArticleExists(article_id)) {
```

Comment:
I'm not sure if this is needed here - as it will be a psql error if the article_id does not exist?